WordPress/src/main/res/layout/login_magic_link_request_screen.xml

```diff
@@ -1,73 +1,79 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
```

Comment:
I don't think the overscroll animation is necessary on this screen. Could you add 
`android:overScrollMode="never"` here?